python/_brotli.cc

```diff
@@ -88,6 +87,37 @@ static int lgblock_convertor(PyObject *o, int *lgblock) {
   return 1;
 }
 
+static BROTLI_BOOL compress_stream(BrotliEncoderState* enc, BrotliEncoderOperation op,
+                                   std::vector<uint8_t>* output, uint8_t* input, size_t input_length) {
+  BROTLI_BOOL ok = BROTLI_TRUE;
+
+  while (ok) {
+    size_t available_in = input_length;
+    const uint8_t* next_in = input;
+    size_t available_out = 0;
+
+    ok = BrotliEncoderCompressStream(enc, op,
+                                     &available_in, &next_in,
+                                     &available_out, NULL, NULL);
```

Comment:
_comment copied from previous commit_

I pushed a new temporary commit using `BrotliEncoderTakeOutput`, but I'm getting errors when trying to execute sequences of operations.

Here are some scenarios:
```
process("one two three ")
finish()
```
**success**

```
process("one ")
process("two ")
process("three ")
finish()
```
**success**

```
process("one ")
process("two ")
flush()
process("three ")
```
**failure:** `BrotliEncoderCompressStream` returns `false`

```
process("one ")
process("two ")
process("three ")
flush()
finish()
```
**failure:** `BrotliEncoderIsFinished` returns `false` and output is incomplete

So it seems to me that either
1. something with `BrotliEncoderCompressStream(..., BROTLI_OPERATION_FLUSH, ...)` and `BrotliEncoderTakeOutput` is not working correctly
2. I'm calling `BrotliEncoderCompressStream` incorrectly now.